### DISCUSSION TITLE
Add hyperlink to help navigate in datagrid control.

### DISCUSSION THREAD

**Discussion by ApoorvaPetkar** [unoplatform/uno#9661] [Q&A]
Hi, 

I'm using `    xmlns:controls="using:CommunityToolkit.WinUI.UI.Controls"` for Datagrid control but I'm not able to find any property that I can hyperlink to the datagrid column.

![image](https://user-images.githubusercontent.com/34087324/187374058-2de0e8f4-b157-4e9b-965a-4de31199511f.png)

Can anyone please help me how I achieve it. 
Thanks.

**Comment by jeromelaban** [comment]
As this control comes from the [Windows Community Toolkit](https://github.com/CommunityToolkit/WindowsCommunityToolkit), it's probably best to ask the question there. You can find the sources for DataGrid [here](https://github.com/CommunityToolkit/WindowsCommunityToolkit/tree/main/Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid).
